services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/CableReportServiceAllu.kt

```diff
@@ -97,91 +106,102 @@ class CableReportServiceAllu(
         builder.part("file", ByteArrayResource(file), MediaType.APPLICATION_PDF).filename("file")
         val multipartData = builder.build()
 
-        webClient.post()
-                .uri(baseUrl + "/v2/applications/{applicationId}/attachments", applicationId)
-                .contentType(MediaType.MULTIPART_FORM_DATA)
-                .accept(MediaType.APPLICATION_JSON)
-                .headers { it.setBearerAuth(token) }
-                .body(BodyInserters.fromMultipartData(multipartData))
-                .exchange()
-                .blockOptional()
-                .orElseThrow()
+        webClient
+            .post()
+            .uri("$baseUrl/v2/applications/{applicationId}/attachments", applicationId)
```

Comment:
Question (for the sake of consistency): Would it be possible to use variables in URI in a consistent manner?
* line 90: `"$baseUrl/v2/cablereports/$applicationId"`
* line 111 (and other similar): `"$baseUrl/v2/applications/{applicationId}/attachments", applicationId`